turbopack/crates/turbo-tasks-backend/Cargo.toml

```diff
@@ -38,6 +39,7 @@ smallvec = { workspace = true }
 tokio = { workspace = true }
 tokio-scoped = "0.2.0"
 tracing = { workspace = true }
+thread_local = { version = "1.1.8" }
```

Comment:
Oh, wow, this lib is from the guy who made the rust `parking_lot` library. I'll trust they know what they're doing.